File: tools/api_test_module_c#/SEDAPITESTINGMODULE/Models/Home/Categories.cs
using Newtonsoft.Json;

namespace Models
{
    public class Categories
    {
        [JsonProperty("ID")]
        public int ID { get; set; }

        [JsonProperty("Name")]
        public string? Name { get; set; }

        [JsonProperty("Image")]
        public string? Image { get; set; }
    }
}

File: SED-RestfulApi/api/models/categories.js
const mongoose = require('mongoose');

const categoriesSchema = mongoose.Schema({
    _id:{
        type:Number,
        required:true
    },
    Name:{
        type:String,
        required:true
    },
    Image:{
        type:String,
        required:true
    }
});


module.exports = mongoose.model('categories', categoriesSchema);